SimpleChatChallenge/Utils/Serialization.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  Serialization.swift
+//  SimpleChatChallenge
+//
+//  Created by Miguel Angel Askar Rodriguez on 28/07/21.
+//
+
+import Foundation
+
+struct Serialization {
+    static func load<T: Codable>(fromJSON resource: String) -> T? {
```

Comment:
good use of generics, have you read about them?

Reply:
Actually, the credit for this code goes to my mentor, Camilo. The code is his, but I made sure I understood.